api/tests/Behat/bootstrap/v2/Registration/IngestTrait.php

```diff
@@ -808,4 +832,28 @@ public function namedDeputyForCaseNumberShouldHaveAddress(string $caseNumber, st
             'Comparing address defined in step against actual named deputy address'
             );
     }
+
+    private function assertOnInfoMessage(string $expectedMessage)
+    {
+        $flashDiv = $this->getSession()->getPage()->find('css', 'div.opg-alert--info');
+
+        if (is_null($flashDiv)) {
+            $missingDivMessage = <<<MESSAGE
+A div with the class opg-alert--info was not found.
+This suggests one of the following:
+
+- the information flash alert was not triggered
+- the form error appears in a non-GDS error element
+- the flash error element is not using macro.notification.
+MESSAGE;
+
+            throw new BehatException($missingDivMessage);
+        }
+
+        $infoMessageFound = str_contains($flashDiv->getHtml(), $expectedMessage);
+
+        if (!$infoMessageFound) {
+            throw new BehatException(sprintf('The error summary did not contain the expected error message. Expected: "%s", got (full HTML): %s', $errorMessage, $errorHtml));
+        }
+    }
 }
```

Comment:
I think `AssertTrait::assertOnAlertMessage()` may work for this OR could be tweaked slightly to save repetition?

Reply:
I saw the method for errors, and for whatever reason just assumed we weren't doing this for the alerts!

`AssertTrait::assertOnAlertMessage()` worked perfectly!